examples/material_gallery/lib/gallery/drawer.dart

```diff
@@ -3,23 +3,24 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
-import 'package:flutter/scheduler.dart' show timeDilation;
-import 'package:flutter/widgets.dart';
-
-import 'app.dart';
 
 class GalleryDrawer extends StatelessWidget {
-  GalleryDrawer({ Key key }) : super(key: key);
-
-  void _changeTheme(BuildContext context, bool value) {
-    GalleryApp.of(context).lightTheme = value;
+  GalleryDrawer({
+    Key key,
+    this.theme,
+    this.onThemeChange,
```

Comment:
Perhaps this should be called onThemeChanged, since its going to be the value of onChanged